README.md

```diff
@@ -10,7 +11,7 @@ Currently, Spektate consists of a command line interface and a simple dashboard
 
 ## Onboard a Bedrock project to use Spektate
 
-Follow the steps in this [guide](https://github.com/CatalystCode/spk/blob/master/docs/service-introspection-onboarding.md) to onboard a project to use Spektate.  
+Follow the steps in this [guide](https://github.com/CatalystCode/spk/blob/master/docs/service-introspection-onboarding.md) to onboard a project to use Spektate.
```

Comment:
url should be `https://github.com/CatalystCode/spk/blob/master/guides/service-introspection-onboarding.md`